pytest_fixtures/component/taxonomy.py

```diff
@@ -20,6 +20,20 @@ def default_location(session_target_sat):
     return session_target_sat.api.Location().search(query={'search': f'name="{DEFAULT_LOC}"'})[0]
 
 
+@pytest.fixture
+def current_sat_org(target_sat):
+    """Return the current organization assigned to the Satellite host"""
+    sat_host = target_sat.api.Host().search(query={'search': f'name={target_sat.hostname}'})[0]
+    return sat_host.organization.read().name
+
+
+@pytest.fixture
+def current_sat_location(target_sat):
+    """Return the current location assigned to the Satellite host"""
+    sat_host = target_sat.api.Host().search(query={'search': f'name={target_sat.hostname}'})[0]
+    return sat_host.location.read().name
```

Comment:
Nice, I think instead of fixture we can add these to satellie_mixins to call from sat_object similar to the nailgun_proxy property (non-blocking)

And Ideally, sat should be in default org and default loc, if it's not then it could possibly be a bug I assume, so can we add an assertation here as well to check if its default org and loc always if we don't assume?

Reply:
Oh @ogajduse merged already, @pnovotny maybe followup PR can be used if this suggestion make sense

Reply:
@Gauravtalreja1 good point, I'll create new PR and move the functionality from the fixtures to sat_mixins as you suggested.